evap/student/templates/student_vote.html

```diff
@@ -30,16 +32,39 @@
     {% endif %}
     {% if preview and not for_rendering_in_modal %}
         <small>
-        <div class="bs-callout bs-callout-info">
-            <b>{% trans 'Questionnaire Preview' %}</b><br />
-            {% blocktrans %}This is a preview of the questionnaire for the course. Students will see the questions below when participating in the evaluation.{% endblocktrans %}
-        </div>
-    </small>
+            <div class="bs-callout bs-callout-info">
+                <b>{% trans 'Questionnaire Preview' %}</b><br />
+                {% blocktrans %}This is a preview of the questionnaire for the course. Students will see the questions below when participating in the evaluation.{% endblocktrans %}
+            </div>
+        </small>
     {% endif %}
     <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
     <form id="student-vote-form" method="POST" class="form-horizontal">
         {% csrf_token %}
 
+        {% if not preview %}
```

Comment:
showing a disabled button in the card on the preview page should be easy. or would you like it to be functional as in opening the card and showing the dropout questionnaire?

Reply:
i think the minimum is that the page *looks* exactly like it looks to students, that's kind of the point of a preview.

whether or not the page should *work* exactly like it does to students, i'm not sure... in this case it might be nice to actually show the dropout questionnaire. if you think of it as not really doing anything but rather showing another part of the page, it's a part of the preview. however, i don't have strong feeling there and if it's too much work, we can postpone that or leave it out.